package.json

```diff
@@ -233,7 +233,7 @@
     "text-loader": "^0.0.1",
     "url-search-params": "^0.10.0",
     "us-forms-system": "https://github.com/usds/us-forms-system.git#84887dc7cb0ccee2ae2c8156b5664697c2850b1d",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#4e5d6a1082e8268b79cb3c433ba379ce97343bc0",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#223ad037ab294b946079e351a5356c9d678b12a0",
```

Comment:
The latest version is now: 444e3aba11ac1ce27e869d24bbee20160bfa3ea6, let's make sure when we merge this that the diff indicates that we are updating the version from 4e5d6a1082e8268b79cb3c433ba379ce97343bc0 and not switching it back from 444e3aba11ac1ce27e869d24bbee20160bfa3ea6 if it has already been updated by another PR.